src/server/engine_shard_set.cc

```diff
@@ -39,6 +52,8 @@ using absl::GetFlag;
 
 namespace {
 
+constexpr DbIndex DEFAULT_DB_INDEX = 0;
+
```

Comment:
In theory shard access with the on idle task is always exclusive relative to regular tasks? Or is it not? Maybe it should be pointed out somewhere in the comments